lib/internal/util/inspect.js

```diff
@@ -1913,8 +1913,11 @@ function reduceToSingleString(
         const start = output.length + ctx.indentationLvl +
                       braces[0].length + base.length + 10;
         if (isBelowBreakLength(ctx, output, start, base)) {
-          return `${base ? `${base} ` : ''}${braces[0]} ${join(output, ', ')}` +
-            ` ${braces[1]}`;
+          const joinedOutput = join(output, ', ');
+          if (!joinedOutput.includes('\n')) {
```

Comment:
```suggestion
          if (!StringPrototypeIncludes(joinedOutput, '\n')) {
```

Reply:
Thanks for the suggestions. However, I kindly ask not to suggest to add more primordials up until there is a decision how to progress with primordials in general.

Reply:
Until there's a decision, shouldn't the status quo be continued?